plinth/__main__.py

```diff
@@ -169,7 +169,7 @@ def configure_django():
         'root': {
             'handlers': ['console', 'file'],
             'level': 'DEBUG' if cfg.debug else 'INFO'
-            }
+            }            
```

Comment:
Sorry, this was a blunder. I forgot to save the file after adding the loggers debug information. Added it in the latest commit.